app/parse.py

```diff
@@ -13,10 +47,131 @@ class Product:
     price: float
     rating: int
     num_of_reviews: int
+    # additional_info: dict :TODO test does not cover it
+
+
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+_driver: WebDriver | None = None
```

Comment:
create class and move all functions to them, in this case you can store driver as attribute